allure-generator/src/main/javascript/blocks/pane/styles.scss

```diff
@@ -12,6 +12,10 @@
 
   &__title {
     padding: 0 15px 10px;
+    width:100%;
+    display: table;
```

Comment:
Could we use `display: flex;` here? It would simplify the underlying code.

Reply:
of course! i couldn't find information about browser support for report, so did it in oldschool way)